Source/items.cpp

```diff
@@ -3741,14 +3757,14 @@ void ProcessItems()
 	for (int i = 0; i < ActiveItemCount; i++) {
 		int ii = ActiveItems[i];
 		auto &item = Items[ii];
-		if (!item._iAnimFlag)
+		if (!item._iAnimFlag) // Item isn't being dropped, so don't animate frames

```

Comment:
I would rather just rename this property to `isAnimated`
```suggestion
		if (!item._iAnimFlag) 
```